### PR TITLE
TestBoston app

### PR DESCRIPTION
**Pull request by fat23cat**
Skeleton for TestBoston app.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -398,6 +398,8 @@ jobs:
           study_key: angio
       - conditionally-launch-build-and-store:
           study_key: mbc
+      - conditionally-launch-build-and-store:
```
Comment (zyme) [inline .circleci/config.yml:413]: Soooo awesome that there is zero GCP docker provisioning for this new study :)

File: config/pepperConfig.js.ctmpl [hunk 2]
```diff
@@ -105,4 +105,20 @@ var DDP_ENV = {
   {{end}}
 {{end}}
 
+{{if eq $study_guid "testboston"}}
```
Comment (zyme) [inline config/pepperConfig.js.ctmpl:109]: @fat23cat you  read my mind--ignore my  comment in the backend PR.  I see you've already handled it :)

**Review by zyme — DISMISSED** [review]
LGTM

File: config/pepperConfig.js.ctmpl (hunk 2 above)
Comment (yufengwng) [reply config/pepperConfig.js.ctmpl:109]: @fat23cat I have the study _guid_ as uppercase `TESTBOSTON` in the study-server branch https://github.com/broadinstitute/ddp-study-server/pull/192. Should I change it to lowercase?

Comment (fat23cat) [reply config/pepperConfig.js.ctmpl:109]: @zyme, should we use uppercase or lowercase for the study guid? For now, other study guids names register is not consistent.

**Comment by zyme** [comment]
Let’s go with lower case from here on out.

File: config/pepperConfig.js.ctmpl (hunk 2 above)
Comment (yufengwng) [reply config/pepperConfig.js.ctmpl:109]: Lowercase it is! I'll change in other PR.

**Review by zyme — DISMISSED** [review]

File: config/pepperConfig.js.ctmpl [hunk 3]
```diff
@@ -105,4 +105,20 @@ var DDP_ENV = {
...
+    {{if eq $environment "dev"}}
+        DDP_ENV['auth0ClientId'] = "fixme-not-set";
```
Comment (yufengwng) [inline config/pepperConfig.js.ctmpl:119]: I have added an auth0 client in `dev`. Can you look in vault or in auth0 dashboard and use that? Should be the one with name `testboston-app-client`.

Comment (fat23cat) [reply config/pepperConfig.js.ctmpl:119]: @yufengwng. I don't see `testboston-app-client` in https://manage.auth0.com/dashboard/us/ddp-dev/applications

Comment (fat23cat) [reply config/pepperConfig.js.ctmpl:119]: Ah, it should be another tenant?

Comment (fat23cat) [reply config/pepperConfig.js.ctmpl:119]: Could you please invite me? The Switch tenant button is inactive for me.
image

Comment (fat23cat) [reply config/pepperConfig.js.ctmpl:119]: 65401f4276d956f165d4088c2260e84a4a5fd2d8

File: config/manifest-study.rb [hunk 4]
```diff
@@ -1,5 +1 @@
-render "docker-compose-study.yaml.ctmpl"
 render "pepperConfig.js.ctmpl"
-copy_file "study-nginx.conf"
-copy_secret_from_path "secret/pepper/#{$env}/#{$version}/#{$study_key}/cert.key"
-copy_secret_from_path "secret/pepper/#{$env}/#{$version}/#{$study_key}/cert.crt"
```
Comment (MocanaAtBroad) [inline config/manifest-study.rb:5]: @zyme We can say bye to this stuff now, yes?

File: config/pepperConfig.js.ctmpl [hunk 5]
```diff
@@ -22,87 +22,116 @@ var DDP_ENV = {
...
-    DDP_ENV['basePepperUrl'] = "https://pepper-staging.datadonationplatform.org";
-{{else if eq $environment "prod"}}
-    DDP_ENV['auth0Domain'] = "login.joincountmein.org";
-    DDP_ENV['auth0Audience'] = "datadonationplatform.auth0.com";
-    DDP_ENV['basePepperUrl'] = "https://pepper.datadonationplatform.org";
+{{if ne $study_guid "testboston"}}
+    {{if eq $environment "dev"}}
+        DDP_ENV['auth0Domain'] = "login-dev.joincountmein.org";
+        DDP_ENV['auth0Audience'] = "ddp-dev.auth0.com";
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
```
Comment (MocanaAtBroad) [inline config/pepperConfig.js.ctmpl:29]: @yufengwng testboston Auth0 tenant ready to roll?

Comment (yufengwng) [reply config/pepperConfig.js.ctmpl:29]: Yes, for the most part.

File: config/manifest-study.rb (hunk 4 above)
Comment (zyme) [reply config/manifest-study.rb:5]: Yes. 👋

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**